packages/flutter_app/lib/features/pub_dev_packages/query/pub_dev_packages_page_state_provider.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:pub_dev_api_client/pub_dev_api_client.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+import '../../../package_adaptor/pub_dev_api_client_provider.dart';
+
+part 'pub_dev_packages_page_state_provider.g.dart';
+
+@Riverpod(keepAlive: true)
+class PackageSearchWordState extends _$PackageSearchWordState {
```

Comment:
![badge](https://img.shields.io/badge/review-nits-inactive)
I would like Doc comments for Public member👌

Would it be better to use `pubDevPackage` here too?